js/app.js

```diff
@@ -6,36 +6,11 @@ window.addEventListener('DOMContentLoaded', function() {
 
   'use strict';
 
-  var ACCESS_TOKEN_KEY = 'access_token';
-  var TOKEN_VALIDITY_KEY = 'token_validity';
-  // Enter a client ID for a web application from the Google Developer Console.
-  // The provided clientId will only work if the sample is run directly from
-  // https://google-api-javascript-client.googlecode.com/hg/samples/authSample.html
-  // In your Developer Console project, add a JavaScript origin that corresponds to the domain
-  // where you will be running the script.
-  var clientId = '265634177893-rejd6a9m1d2q1g5a4pu1tive751g4akm.apps.googleusercontent.com';
-
-  // Enter the API key from the Google Develoepr Console - to handle any unauthenticated
-  // requests in the code.
-  // The provided key works for this sample only when run from
-  // https://google-api-javascript-client.googlecode.com/hg/samples/authSample.html
-  // To use in your own application, replace this API key with your own.
-  var apiKey = 'AIzaSyDwIbLCByl-fhQ6QVYvKBqMsV4h2hueHq4';
-
-  // To enter one or more authentication scopes, refer to the documentation for the API.
-  var scopes = 'https%3A%2F%2Fwww.google.com%2Fm8%2Ffeeds';
-
-  var oauthWindow;
-  var accessToken;
-  var tokenValidity;
-
   var authorizeButton = document.getElementById('authorize-button');
-  authorizeButton.onclick = function(e) {
-    var url = `https://accounts.google.com/o/oauth2/auth?scope=${scopes}` +
-      `&redirect_uri=https%3A%2F%2Fphoxygen.eu%2Foauth_result&response_type=token` +
-      `&client_id=${clientId}&approval_prompt=force&state=friends`;
-    oauthWindow = window.open(url, '', 'dialog');
+  authorizeButton.onclick = () => {
+    OauthManager.start().then(enableImport).catch(disableImport);
```

Comment:
If you capitalize it, then let’s call it `OAuthManager` please — see http://oauth.net/